source/community/release_notes/sumac/customizing_pages.rst

```diff
@@ -0,0 +1,8 @@
+Customizing Site Header and Learner Dashboard
+#############################################
+
+Coming soon in Sumac: leverage _frontend plugin slots_ to quickly and easily
+modify visual elements on the site header and on the Learner Dashboard. Be sure
+to watch this page for more detail!
```

Comment:
@crathbun428 - do you think it's reasonable to merge in a bit of hype for the upcoming release? 😀

Reply:
Yes - this looks great!